packages/arco-lib/src/components/Table/Table.tsx

```diff
@@ -172,7 +172,7 @@ export const Table = implementRuntimeComponent({
     state: TableStateSpec,
     methods: {},
     slots: {
-      content: {
```

Comment:
Although `td` is more proper than `content`, it is a breaking change. We'd better avoid breaking changes, before we have upgrade tool.

Reply:
ok,I have changed back to content